experiments/long_runs/leaderboard/leaderboard.jl

```diff
@@ -0,0 +1,165 @@
+import ClimaAnalysis
+import GeoMakie
+import CairoMakie
+import Dates
+
+include("data_sources.jl")
+
+"""
+    compute_leaderboard(leaderboard_base_path, diagnostics_folder_path)
+
+Plot the biases against observations and simulation data.
```

Comment:
I see that both obs_var_dict and sim_var_dict are used in this function but not passed in as arguments. For clarity and to make it easy on future users, could we pass in those as arguments?

Reply:
Is this argument used? diagnostics_folder_path? Why pass this if we also have already created the sim_var_dict?

Reply:
I am going to make the entire `data_sources.jl` file a function instead since I realized it is confusing how things are being initialized before.